src/openrct2/network/Network.cpp

```diff
@@ -1948,6 +1948,9 @@ void Network::ProcessPlayerInfo()
 
 void Network::ProcessGameCommands()
 {
+    // Remove ghost scenery so it doesn't interfere with incoming network command
+    scenery_remove_ghost_tool_placement();
```

Comment:
This should be moved into L1982 and protected by an if gc->GetType() == GAME_COMMAND_PLACE_WALL, LARGE_SCENERY, BANNER... 

At present it just makes ghosts redraw every tick which makes ghosts that have some random element (such as rotation) constantly change.